src/sentry/notifications/manager.py

```diff
@@ -235,6 +235,26 @@ def get_for_users_by_parent(
             target__in=[user.actor.id for user in users],
         )
 
+    def get_for_team(
+        self,
+        type: NotificationSettingTypes,
+        parent: Any,
+        team: Any,
+    ) -> QuerySet:
+        """
+        Find all of a project/organization's notification settings for a team.
+        This will include each team's project/organization-independent settings.
+        """
+        scope_type = get_scope_type(type)
```

Comment:
For now this will always be issue alerts, but I figured I'd leave it open for other types in the future if we do that.